client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -1,15 +1,17 @@
-import { Card } from '@automattic/components';
+import { Card, ScreenReaderText } from '@automattic/components';
 import { ToggleControl } from '@wordpress/components';
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import { PureComponent } from 'react';
 import { connect } from 'react-redux';
+import FormLabel from 'calypso/components/forms/form-label';
 import { CHANGE_NAME_SERVERS } from 'calypso/lib/url/support';
 import {
 	composeAnalytics,
 	recordGoogleEvent,
 	recordTracksEvent,
 } from 'calypso/state/analytics/actions';
+const NAMESERVERS_TOGGLE_TITLE = 'Use WordPress.com Name Servers';
 
 class NameserversToggle extends PureComponent {
```

Comment:
Regarding the previous comment: 

Since your component is already wrapped in the `localize` HOC, I would move the previous title `const` declaration to the instance level (constructor) and use the `translate` prop. Something like: 

```suggestion
class NameserversToggle extends PureComponent {
    constructor( props ){
        super( props );

        this.NAMESERVERS_TOGGLE_TITLE = props.translate( 'Use WordPress.com Name Servers' );
    }
```

Reply:
I don't see the need to do this either. Avoiding calling a `translate()` again brings little to no performance improvement, at the cost of a new variable, so I don't think we need to do it.

However I agree that the component already has access to the `translate` prop and can freely use it.